packages/@okta/vuepress-theme-prose/util/coveo/templates.js

```diff
@@ -0,0 +1,48 @@
+import config from "./config.js";
+
+const registerTemplates = () => {
+  Coveo.TemplateHelpers.registerTemplateHelper("sourceTag", function(source) {
+    let markup = "";
+    let commonSource = this.result.raw.commonoktasource[0];
+    let customCaption = config.facetValueCaptions[commonSource];
```

Comment:
Any reason not to use `const` for the values that are not modified?

Reply:
Copy pasted most of this code ;) I'll give it a look